core/src/executor/aggregate/state.rs

```diff
@@ -159,7 +156,7 @@ pub struct State<'a> {
     group: Group,
     values: IndexMap<(Group, &'a Aggregate), (usize, AggrValue)>,
     groups: HashSet<Group>,
-    contexts: Vector<Rc<BlendContext<'a>>>,
+    contexts: Vector<Rc<RowContext<'a>>>,
```

Comment:
The `Context` type, which is only used in this file, is currently used only once.
What about eliminating or unifying?